### ISSUE TITLE
EU4 Peasant Republics converting to Absolute Monarchy in V2

### ISSUE THREAD

**Issue by BFKelleher** [Idhrendur/paradoxGameConverters#313]
Not sure how it's happening, but here's the history file of an early game Dithmarschen
[DTT - Dithmarschen.txt](https://github.com/Idhrendur/paradoxGameConverters/files/1150675/DTT.-.Dithmarschen.txt)

**Comment by kratostatic** [comment]
Yeah, for whatever reason the converter doesn't like to create a Proleteriat Dictatorship (which is what we're currently trying to convert Peasant's Republics as) so it defaults to Absolute Monarchy.  I hadn't bothered looking into this in too much detail because frankly Dithmarschen will normally disappear in most games by the time they are converted.  I think I want to change a few other things as well - for example they should have a national order of equality (or maybe liberty) rather than order.

A quick fix for your game would be to change the government mapping (probably to Democracy) and re-run the conversion.

**Comment by kratostatic** [comment]
I _think_ this is fixed by 0c0304b4d9619f754b4e1c31a17136291cd4b5ff but I will leave this open until I can confirm.
